Add Catalogue for Software Management Plan (#165)
rdmorganiser/rdmo-catalog: shared/Max-Planck-Digital-Library/Software-Management-Plan/documentation.md

# Development
This catalogue was developed through many conversations and iterations. It represents an attempt or approximation of what a Software Management Plan could possibly be. The main focus from our side was on added value for scientists who write software for their research themselves, but have not learned how to do this or have learned it only a little. At the same time, such an SMP can also be helpful in advising information and infrastructure experts.

Furthermore, this SMP template was inspired by some documentations, specially:
* https://www.software.ac.uk/resources/guides/software-management-plans
* https://opencarp.org/about/software-management-plan
* https://zenodo.org/record/7248877
* https://biohackrxiv.org/k8znb/

# Implementing
The catalogue uses some of the generic elements from RDMO. Therefore, all generic elements from "Conditions", "Domain", "Options" and "Questions" must already be implemented in advance.

> To use this SMP, the additional SMP elements "Condition", "Domains" and "Questions" must then be imported into the own RDMO system. The respective order can be found in the generic readme of the RDMO and must be observed.

Reviewer: ```suggestion
To use this SMP, the additional SMP elements "Condition", "Domain" and "Questions" must then be imported into the own RDMO system. The respective order is documented in the [generic Readme file of the RDMO content](https://github.com/rdmorganiser/rdmo-catalog/tree/master/rdmorganiser#readme) and must be observed.
```

# Transferability

> This catalogue uses many custom domains, as the generic RDMO elements do not always cover the specific aspects of software. For this reason, the existing views in RDMO cannot be used for this catalogue, or only to a very limited extent.

Reviewer: ```suggestion
This catalogue uses many custom attributes, as the generic RDMO attributes do not always cover the specific aspects of software. For this reason, the existing views in RDMO cannot be used for this catalogue, or only to a very limited extent.
```
